microsoft-edge/extensions-chromium/developer-guide/native-messaging.md

```diff
@@ -38,13 +38,11 @@ Add the nativeMessaging permission to the extension’s **manifest.json** file.
               "128": "icon-128.png"}, 
           "permissions": ["nativeMessaging"] 
     }
-```
+```  
 
-### Step 2: Create your native messaging host manifest file
+## Step 2 - Create your native messaging host manifest file  
     
-Native applications must provide a native messaging host manifest file. This manifest file contains the path to the native messaging host executable, the method of communication with the extension, and a list of allowed extensions it can communicate with. Browsers read and validate the native messaging host manifest, but it’s never installed or managed by the browser. 
-The host manifest file must be a valid json file containing the following fields.
-
+Native applications must provide a native messaging host manifest file.  the manifest file contains the path to the native messaging host runtime, the method of communication with the extension, and a list of allowed extensions to which it communicates.  The browser reads and validates the native messaging host manifest.  The browser does not install or manage the manifest file.  
```

Comment:
Native applications must provide a native messaging host manifest file.  The manifest file contains the path to the native messaging host runtime, the method of communication with the extension, and a list of allowed extensions to which it communicates.  The browser reads and validates the native messaging host manifest.  The browser does not install or manage the native messaging host manifest file.

Reply:
Updated.

```md
Native applications must provide a native messaging host manifest file.  The manifest file contains the path to the native messaging host runtime, the method of communication with the extension, and a list of allowed extensions to which it communicates.  The browser reads and validates the native messaging host manifest.  The browser does not install or manage the native messaging host manifest file.  
```